be/src/common/configbase.cpp

```diff
@@ -229,9 +230,17 @@ Status set_config(const std::string& field, const std::string& value) {
     if (!it->second->set_value(value)) {
         return Status::InvalidArgument(fmt::format("Invalid value of config '{}': '{}'", field, value));
     }
+    update_config(field, value);
```

Comment:
what's the expected way to trigger this config update?

Reply:
configure a dynamic parameter of a BE node by using the curl command.
curl -X POST http://{be_ip}:{be_http_port}/api/update_config?sys_log_level=ERROR

Reply:
check the `UpdateConfigAction::update_config` implementation, there is a update callback hook that processes the necessary update of configvar. shall keep the same way as other dynamic mutable vars.